src/features/surveys-slice.ts

```diff
@@ -33,7 +33,7 @@ export const fetchSurveys = createAsyncThunk<Survey[], void, { state: RootState;
         throw new Error(strings.errorHandling.missingAccessToken);
       }
 
-      return await Api.getSurveysApi(keycloak.token).listSurveys({});
+      return await Api.getSurveysApi(keycloak.token).listSurveys({ maxResults: 2147483647 });
```

Comment:
This is meant to be maximum int32 value, right? It could be moved to constant with such name, like MAX_INT_32. Then it's easier to understand.

```suggestion
      const INT_32_MAX_VALUE = 2147483647;
      return await Api.getSurveysApi(keycloak.token).listSurveys({ maxResults: INT_32_MAX_VALUE });
```